### PR TITLE
Update turn-audit-log-search-on-or-off.md

### PR DESCRIPTION
**Pull request by msbemba**
made changes to the navigation in the compliance center to turn on the Audit log.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5016

File: microsoft-365/compliance/turn-audit-log-search-on-or-off.md [hunk 1]
```diff
@@ -52,17 +52,13 @@ If audit log search is not turned on for your organization, you can turn it on i
...
 
-   If audit log search is not turned on for your organization, a banner is displayed saying that auditing has to be turned on to record user and admin activity.
+   If audit log search is not turned on for your organization, a banner is displayed saying **Start recording user and admin activity**.
 
-3. Click **Turn on auditing**.
-
-    ![Click Turn on auditing](../media/39a9d35f-88d0-4bbe-a962-0be2f838e2bf.png)
-  
-    The banner is updated to say the audit log is being prepared and that you can search for user and admin activity in a few hours.
+3. Click  **Start recording user and admin activity**. You will be prompted to confirm if **Your Oraganization settings need to be udpated. Do you want to Continue?**. Click **Yes** to turn on Audit log search.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/turn-audit-log-search-on-or-off.md]:
```suggestion
3. Select**Start recording user and admin activity**. You will be prompted to confirm if **Your Organization settings need to be updated. Do you want to Continue?**. Select **Yes** to turn on audit log search.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

File: microsoft-365/compliance/turn-audit-log-search-on-or-off.md [hunk 2]
```diff
@@ -52,17 +52,13 @@ If audit log search is not turned on for your organization, you can turn it on i
...
 
-   If audit log search is not turned on for your organization, a banner is displayed saying that auditing has to be turned on to record user and admin activity.
+   If audit log search is not turned on for your organization, a banner is displayed saying **Start recording user and admin activity**.
 
-3. Click **Turn on auditing**.
-
-    ![Click Turn on auditing](../media/39a9d35f-88d0-4bbe-a962-0be2f838e2bf.png)
-  
-    The banner is updated to say the audit log is being prepared and that you can search for user and admin activity in a few hours.
+3. Select**Start recording user and admin activity**. You will be prompted to confirm if **Your Organization settings need to be updated. Do you want to Continue?**. Select **Yes** to turn on audit log search.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/turn-audit-log-search-on-or-off.md]:
```suggestion
3. Select **Start recording user and admin activity**. You will be prompted to confirm if **Your Organization settings need to be updated. Do you want to Continue?**. Select **Yes** to turn on audit log search.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba - one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@markjjo Please merge this PR. Thanks

**Pull request merged**